Pythonwin/pywin/framework/sgrepmdi.py

```diff
@@ -637,9 +639,15 @@ def OnOK(self):
 
 
 class GrepParamsDialog(dialog.Dialog):
-    def __init__(self, items):
-        self.items = items
-        self.newitems = []
+    def __init__(
+        self,
+        # Sequence rather than Iterable to avoid single-use iterable
+        items: Sequence[str],
+    ):
+        # This subclass breaks LSP by making "items" not a method
+        # TODO: Consider making this a method or changing the base class to a property
+        self.items: Sequence[str] = items  # type: ignore[assignment]
```

Comment:
This is revealing an easy source of bugs where some Dialogs use `.items()` and some Dialogs use `.items`